concent_api/core/exceptions.py

```diff
@@ -1,11 +1,19 @@
+from utils.constants import ErrorCode
+
 
 class UnexpectedResponse(Exception):
     pass
 
 
-class Http400(Exception):
+class ConcentInSoftShutdownMode(Exception):
     pass
 
 
-class ConcentInSoftShutdownMode(Exception):
-    pass
+class Http400(Exception):
+
+    def __init__(self, error_message: str, error_code: ErrorCode, *args, **kwargs) -> None:
+        assert isinstance(error_message, (str, None))
```

Comment:
if we want to allow `error_message` to be `None`, then in the method's signature it should be -> error_message: **Optional[str]**

Reply:
:+1: